plugin/storage/memory/memory.go

```diff
@@ -339,3 +335,73 @@ func flattenTags(span *model.Span) model.KeyValues {
 	}
 	return retMe
 }
+
+func hasCorrespondingServerSpan(clientSpan *model.Span, spans []*model.Span) bool {
```

Comment:
```suggestion
func isLeaf(clientSpan *model.Span, spans []*model.Span) bool {
```
isn't "leaf" what you're looking for?

Reply:
Yes, I have updated the function name.